app/app.go

```diff
@@ -449,7 +448,10 @@ func NewApp(
 	)
 	app.evmKeeper = evmkeeper.NewKeeper(
 		appCodec, keys[evmtypes.StoreKey], tkeys[evmtypes.TransientKey], evmSubspace,
-		app.accountKeeper, NewEVMBankKeeper(app.bankKeeper), app.stakingKeeper, app.feeMarketKeeper,
+		NewEVMAuthKeeper(app.accountKeeper, authtypes.ProtoBaseAccount),
```

Comment:
should `ethermint.ProtoAccount` be passed here?

Reply:
yes, updated in https://github.com/Kava-Labs/kava/pull/1177/commits/82e67bb64b2975af089c42aa55622c6e4f46fbe0